Safety/Src/RSSI.cpp

```diff
@@ -4,22 +4,17 @@
 static volatile bool failed = false;
 static volatile int16_t counter = 0;
 
+//Time is in microseconds
+
+constexpr uint8_t MIN_ON_TIME = 1200; //20% of pwm signal from 1000 to 2000 microseconds
```

Comment:
Not sure if I'm ootl on this one, how did we decide 20%

Reply:
I'm also out of the loop. We should justify this number. I don't want to fall out of the sky if we still have a flyable connection, and likewise don't want to keep flying if the pilot cant really control anything. (I believe RSSI from our receiver is measured in dBm from 0 to -130 and so is not a linear scale)